recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
-{% set version = "2.4.7" %}
+{% set version = "2.5.4" %}
+{% set libltdl_version = "2.4.3a" %}
 
 package:
-  name: libtool
+  name: libtool-split
```

Comment:
You should add a `feedstock-name` to `extras`

Reply:
[`0883214` (#35)](https://github.com/conda-forge/libtool-feedstock/pull/35/commits/0883214c2745ae402758b6e24fda63fcf4b8e521)